package.json

```diff
@@ -63,6 +63,11 @@
                 "icon": "src/assets/back.png",
                 "category": "Google Play Music"
             },
+            {
+                "command": "gmusic.getTrackURL",
+                "title": "Get Current Track URL",
```

Comment:
If we're changing the command to not require interacting with the information box, let's change the title to `Copy Track URL to Clipboard`.